cordova-common/spec/CordovaCheck.spec.js

```diff
@@ -30,11 +30,12 @@ describe('findProjectRoot method', function() {
         process.env.PWD = origPWD;
         process.chdir(cwd);
     });
+function removeDir(someDirectory) {
+    shell.rm('-rf', someDirectory);
+}
     it('should return false if it hits the home directory', function() {
         var somedir = path.join(home, 'somedir');
-        this.after(function() {
```

Comment:
Here does this.after remove the somedir similarly to how we are using in for testing save/fetch? In the beforeEach, I have ``` shell.rm('-rf', project); ``` to clear the previous temp project? Maybe I should just use my function in a beforeEach?  But then how will the someDir get updated if it changes from test to test? (if that makes sense)